test/functional/filters/multilineexception/multiline_exception_test.go

```diff
@@ -108,12 +106,34 @@ created by main.main
 			Expect(logs[0].Message).To(Equal(exception))
 		}
 	},
-		Entry("of Java services", javaException, nil),
-		Entry("of JS client side exception", jsClientSideException, nil),
-		Entry("of V8 errors stack trace", jsV8Exception, nil),
-		Entry("of NodeJS services", nodeJSException, nil),
-		Entry("of GoLang services", goLangException, nil),
-		Entry("of single application NS sources with multiple pipelines", goLangException, func(framework *functional.CollectorFunctionalFramework) {
+		// exceptions to stdout
+		Entry("of Java services to stdout stream", constants.STDOUT, javaException, nil),
+		Entry("of JS client side exception to stdout stream", constants.STDOUT, jsClientSideException, nil),
+		Entry("of V8 errors stack trace to stdout stream", constants.STDOUT, jsV8Exception, nil),
+		Entry("of NodeJS services to stdout stream", constants.STDOUT, nodeJSException, nil),
+		Entry("of GoLang services to stdout stream", constants.STDOUT, goLangException, nil),
+		Entry("of single application NS sources with multiple pipelines to stdout stream", constants.STDOUT, goLangException, func(framework *functional.CollectorFunctionalFramework) {
+			b := testruntime.NewClusterLogForwarderBuilder(framework.Forwarder).
+				FromInputName("multiline-log-ns", func(spec *obs.InputSpec) {
+					spec.Type = obs.InputTypeApplication
+					spec.Application = &obs.Application{}
+				}).
+				WithMultilineErrorDetectionFilter().
+				ToHttpOutput()
+			//LOG-2241
+			b.FromInput("multiline-log-ns").
+				Named("other").
+				WithMultilineErrorDetectionFilter().
+				ToElasticSearchOutput()
+		}),
+
+		// exceptions to stderr
+		Entry("of Java services to stderr stream", constants.STDERR, javaException, nil),
```

Comment:
Consider reverting this and only update some of the existing tests to use STDERR.  The logic will suffice and it keeps from expanding the tests unnecessarily